CSS/style.css

```diff
@@ -0,0 +1,284 @@
+html {
+  scroll-behavior: smooth;
+}
+
+:root {
+  --primary: #172b4d;
+  --h-color: #3a4a42;
+  --secondary: #36b37e;
+  --secondary-btn: #36b37f;
+  --weight: bold;
+  --background: #ebf0ee;
+}
+
+.h2-size {
+  color: var(--primary);
+  font-size: 40px;
+  font-weight: var(--weight);
+  line-height: 52px;
+}
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+  overflow-x: hidden;
+}
+
+.d-flex {
+  display: flex;
+}
+
+.flex-center {
+  align-items: center;
+  justify-content: center;
+}
+
+.mobile-invisible {
+  display: block;
+}
+
+.desktop-invisible {
+  display: none;
+}
+
+body {
+  width: 100%;
+  min-height: 100vh;
+  font-family: 'Inter', sans-serif;
+  scroll-behavior: smooth;
+  overflow-x: hidden;
+}
+
+header {
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  padding: 14px;
+}
+
+header h2 {
+  position: relative;
+  animation: crane-float 2s alternate 5 ease-in-out both;
+}
+
+@keyframes crane-float {
+  0% {
+    transform: translateY(0);
+  }
+
+  100% {
+    transform: translateY(-10px);
+  }
+}
+
+nav .nav-menu {
+  visibility: hidden;
+  transform: translateY(-100%);
+}
+
+nav .nav-menu.open,
+nav .nav-menu.hide {
+  visibility: visible;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: flex-start;
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  z-index: 2;
+  transition: all 1s ease-in-out;
+  transform: translateY(0%);
+  width: 100%;
+  height: 100%;
+  padding-left: 28px;
+  padding-top: 62px;
+  background-color: #fff;
+}
+
+nav .nav-menu.hide {
+  visibility: hidden;
+  transform: translateY(-100%);
+}
+
+nav .nav-menu.open .nav-link,
+nav .nav-menu.hide .nav-link {
+  font-size: 32px;
+  line-height: 44px;
+  font-weight: 600;
+  margin: 12px;
+}
+
+nav .nav-menu.open .nav-link a,
+nav .nav-menu.hide .nav-link a {
+  text-decoration: none;
+  color: var(--h-color);
+}
+
+nav .show-menu-btn {
```

Comment:
- [ ] [REQUIRED] As stated in the project requirements, You must stick to the design as much as possible (e.g. font, colors, images, tests, margins) using the templates in Figma. Please reduce the size of the hamburger button to match the original design.😊😊

Reply:
I do the change, I already reduce the size of the hamburger button